dts/bindings/sensor/rohm,bh1730.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2025, Borislav Kereziev
+# SPDX-License-Identifier: Apache-2.0
+
+description: Rohm BH1730 ambient light sensor.
+
+compatible: "rohm,bh1730"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  gain:
+    type: int
+    default: 0
```

Comment:
Please document the default values 
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values